- Remove qlo10k1 package, only works with Qt2/Qt3
- Remove echomixer package, only works with GTK+2.0
- Remove rmedigicontrol package, only works with GTK+2.0

diff --git a/Makefile b/Makefile
index df6a25b..4b64330 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 VERSION = 1.2.15
 TOP = .
 SUBDIRS = as10k1 envy24control hdsploader hdspconf hdspmixer \
-	  mixartloader pcxhrloader rmedigicontrol sb16_csp seq sscape_ctl \
-	  us428control usx2yloader vxloader echomixer ld10k1 qlo10k1 \
+	  mixartloader pcxhrloader sb16_csp seq sscape_ctl \
+	  us428control usx2yloader vxloader ld10k1 \
 	  hwmixvolume hdajackretask hda-verb hdajacksensetest
 
 all:
